pkgs/applications/window-managers/way-cooler/default.nix

```diff
@@ -9,65 +9,67 @@ let
   fakegit = writeShellScriptBin "git" ''
     echo ""
   '';
-  way-cooler = ((callPackage ./way-cooler.nix {}).way_cooler_0_6_2.override {
+  lua = lua5_3.overrideAttrs (oldAttrs: rec {
+    postFixup = ''
+      ln -s $out/lib/liblua.so $out/lib/liblua5.3.so
```

Comment:
This seems weird. Why not just patch lua to do this itself always?

Reply:
@grahamc Fixed, thanks!